File: LeetCode/0004_Median_of_Two_Sorted_Arrays/solution.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function(nums1, nums2) {
var arr = [...nums1,...nums2];
arr.sort((a, b) => a - b);
if(arr.length %2==0) return (arr[(arr.length/2)]+arr[(arr.length/2)-1])/2;
else return arr[(arr.length-1)/2]
};

File: LeetCode/0004_Median_of_Two_Sorted_Arrays/solution.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        vector<int>n = nums1;
        n.insert(n.end(), nums2.begin(), nums2.end());
        sort(n.begin(),n.end());
        if(n.size()%2==0){
            return (float)(n[n.size()/2]+n[n.size()/2 -1])/2;
        }
        return n[n.size()/2];
    }
};